src/App.css

```diff
@@ -40,4 +40,10 @@ border-bottom-width: 1px;
   border-width: 1px;
   color: orange;
   font-weight: bolder;
+}
+.registerlogolocation{
```

Comment:
Name of few words needs to be "CamelCase"
That means the first letter of a new word is capital (However the first word still lower case)
registerLogoLocation